starter-core/src/main/java/io/micronaut/starter/feature/view/JTE.java

```diff
@@ -80,6 +81,18 @@ public void apply(GeneratorContext generatorContext) {
         generatorContext.addTemplate("exampleJte", new RockerTemplate(JTE_SRC_DIR + "/example.jte", exampleJTE.template()));
     }
 
+    protected void addDependencies(GeneratorContext generatorContext) {
+        generatorContext.addDependency(MicronautDependencyUtils.viewsDependency()
+                .artifactId(ARTIFACT_ID_MICRONAUT_VIEWS_JTE)
+                .compile());
+        if (generatorContext.getLanguage() == Language.KOTLIN) {
```

Comment:
Why are both dependencies with a scope compile? I would expect the view dependency as `implementation` and the other as `compileOnly` - in Gradle terminology

Reply:
I changed it to compileOnly in the module https://github.com/micronaut-projects/micronaut-views/pull/877

Reply:
> Are you sure we need to have that dependency in Micronaut Views?

What dependency?

Reply:
You wrote that you changed it to `compileOnly`

Reply:
Regarding this PR, it's a bit strange to have micronaut-views and kotlin with the same scope